ci-operator/config/stolostron/policy-collection/stolostron-policy-collection-main__opp-ocp4.11-interop.yaml

```
@@ -0,0 +1,52 @@
+base_images:
+  cli:
+    name: "4.11"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- dockerfile_literal: |
+    FROM this-is-ignored
+    RUN dnf install -y git
+  from: cli
+  to: cli-with-git
+releases:
+  initial:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.11"
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.11"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: opp-interop-aws
+  interval: 960h
+  steps:
+    cluster_profile: aws-interop-qe
```

```suggestion
    cluster_profile: <another-cluster-profile>
```

@ascerra This cluster profile should be removed before being merged

---

There was a mix up here :). Thought this profile was related to our team @ascerra but ours is `aws-cspi-qe`. Saw interop and thought CSPI. Please ignore!